src/PythonClient/src/quixstreams/helpers/exceptionconverter.py

```diff
@@ -0,0 +1,18 @@
+from ..exceptions.quixapiexception import QuixApiException
+from ..native.Python.InteropHelpers.InteropUtils import InteropException
+
+
+class ExceptionConverter:
+
+    @staticmethod
+    def raise_from_interop(exception: InteropException):
+        if exception.exc_type == "QuixStreams.Streaming.QuixApi.QuixApiException":
```

Comment:
is it better if use `typeof(QuixApiException).FullName` instead of providing a string?